modules/sdk-core/src/bitgo/enterprise/enterprise.ts

```diff
@@ -29,6 +31,7 @@ export class Enterprise implements IEnterprise {
     if (!_.isString(enterpriseData.name)) {
       throw new Error('enterprise name has to be a string');
     }
+    this._enterprise = enterpriseData;
```

Comment:
This is the same pattern used in the wallet class. It was odd to see this only half completed for this class.